<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMixPaper</class>
 <widget class="QWidget" name="NewMixPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Mix Paper</string>
  </property>
  <widget class="QTabWidget" name="tWPaperCreation">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>641</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add qustions</string>
    </attribute>
    <widget class="QPushButton" name="pBClose">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>560</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBMcq">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1141</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>Add MCQ question</string>
     </property>
     <widget class="QLabel" name="lMcqQuestion">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Question :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEMcqQuestion">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>721</width>
        <height>61</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLabel" name="lNoOfChoises">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No of Choises :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cBNoOfChoises">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>120</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lAnswer1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Answer 1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEAnswer1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lAnswer2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Answer 2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEAnswer2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>170</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lAnswer3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Answer 3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEAnswer3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lAnswer4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>225</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Answer 4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEAnswer4">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>220</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pBAddMcq">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>290</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add MCQ Qustion</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gBEssay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>1141</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Add Essay Question</string>
     </property>
     <widget class="QLabel" name="lEssayQusesion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Question :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEEssayQuestion">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>711</width>
        <height>61</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pBAddEssayQuestion">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>140</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Essay Question</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Marks For Question</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lEEssayMarks">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Create Answers</string>
    </attribute>
    <widget class="QGroupBox" name="gBHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBQuestions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>1111</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QFrame" name="fQuestion1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fQuestion5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>964</x>
       <y>580</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save All And Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBPrivious">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>582</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Privious</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBNext">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>582</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="lPageNo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>590</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Edit Questions</string>
    </attribute>
    <widget class="QGroupBox" name="gBEHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="gBEQuestions">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>1111</width>
       <height>501</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QFrame" name="fEQuestion1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="fEQuestion5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>1110</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lEPageNo">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>590</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBEClose">
     <property name="geometry">
      <rect>
       <x>954</x>
       <y>580</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Save All and  Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBENext">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>580</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBEPrivious">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>580</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Privious</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pBClose</sender>
   <signal>clicked()</signal>
   <receiver>NewMixPaper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1059</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>1203</x>
     <y>596</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>NewMixPaper</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1103</x>
     <y>612</y>
    </hint>
    <hint type="destinationlabel">
     <x>1229</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
